app/workers/fast_track/hypertension_pdf_generator.rb

```diff
@@ -0,0 +1,242 @@
+# frozen_string_literal: true
+
+module FastTrack
+  class HypertensionPdfGenerator
+    attr_accessor :patient, :bp_data, :medications
+
+    def initialize(patient, bp_data, medications, date)
+      @patient = patient
+      @bp_data = bp_data
+      @medications = medications
+      @date = date
+    end
+
+    def generate
+      pdf = Prawn::Document.new
+      pdf = add_intro(pdf)
+      pdf = add_blood_pressure(pdf)
+      pdf = add_medications(pdf) if medications.length > 1
+      add_about(pdf)
+    end
+
+    def stringify_patient
+      suffix = patient[:suffix].present? ? ", #{patient[:suffix]}" : ''
+      stringified = ''
+      %i[first middle last].each do |name|
+        stringified = "#{stringified} #{patient[name]}" if patient[name].present?
+      end
+
+      "#{stringified}#{suffix}"
+    end
+
+    def add_intro(pdf)
+      patient_name = stringify_patient
+      gen_time = Time.now.getlocal
+      gen_stamp = "#{gen_time.strftime('%m/%d/%Y')} at #{gen_time.strftime('%l:%M %p %Z')}"
```

Comment:
Nit: Don't use shorthand for variables and stick to the `_at` convention for time stamps (i.e. `generated_at` instead of `gen_stamp`).

Reply:
thanks, fixed it here: https://github.com/department-of-veterans-affairs/vets-api/pull/8558/commits/989e639c0b6996c4726f43eba865e33aab5e06b6